### PR TITLE
[IOS] Create Home Screen

### PR DESCRIPTION
**Pull request by zhenyapavlovsky**
## What's new
1. Create UI for `Home screen`

## Issue

[Issue](https://trello.com/c/igDtwZY6)

## Screenshot

Screenshot 2022-12-06 at 18 59 18

### CODE REVIEW

File: Crypto_App_iOS/Flows/Screens/Home/HomeView.swift [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+struct HomeView: View {
+    
+    @ObservedObject var viewModel: HomeViewModel
+    
+    var body: some View {
+        NavigationView {
+            ZStack {
+                VStack {
+                    Form {
+                        bitcoinSection
```
Comment (vasyl-nadtochii) [inline Crypto_App_iOS/Flows/Screens/Home/HomeView.swift:19]: please, remove this `vars` and create some dummy array of data. Then, using `ForEach` create list UI

File: Crypto_App_iOS/Flows/Screens/Home/HomeView.swift [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+            }
+        }
+        .listRowBackground(Color.noire)
+        .frame(height: 40)
+    }
+    
+    var buttonSettings: some View {
+        Button(action: {}, label: {
+            Image.settingsButton
+
```
Comment (vasyl-nadtochii) [inline Crypto_App_iOS/Flows/Screens/Home/HomeView.swift:100]: extra space

**Review by vasyl-nadtochii — CHANGES_REQUESTED** [review]

File: Crypto_App_iOS/Flows/Screens/Home/HomeView.swift (hunk 2 above)
Comment (zhenyapavlovsky) [reply Crypto_App_iOS/Flows/Screens/Home/HomeView.swift:100]: Done

File: Crypto_App_iOS/Flows/Screens/Home/HomeView.swift [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+                        .background(Color.carbon)
+                        .cornerRadius(10)
+                        .foregroundColor(Color.cultured)
+                }
+                .fontWeight(.bold)
+            }
+        }
+    }
+    
+//    var bitcoinSection: some View {
```
Comment (vasyl-nadtochii) [inline Crypto_App_iOS/Flows/Screens/Home/HomeView.swift:65]: why do we need to have that commented? If it's not needed, just remove that

File: Crypto_App_iOS/Flows/Screens/Home/Data.swift [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    struct Data: Identifiable {
+        
+        let imageName: Image
+        let nameCoin: String
+        let priceCoin: String
+        let id = UUID()
+        
+    }
+
+    let sectionCoin = [
```
Comment (vasyl-nadtochii) [inline Crypto_App_iOS/Flows/Screens/Home/Data.swift:19]: move this array to the `HomeViewModel`

File: Crypto_App_iOS/Resources/en.lproj/Localizable.strings [hunk 5]
```diff
@@ -11,3 +11,9 @@
 "welcome" = "Welcome";
 "to_the_crypto" = "To the Crypto";
 "let's_start" = "Let's start!";
+
+// Mark: HomeView
+
+"bitcoin" = "Bitcoin";
```
Comment (vasyl-nadtochii) [inline Crypto_App_iOS/Resources/en.lproj/Localizable.strings:17]: I guess we don't need that more

**Review by vasyl-nadtochii — APPROVED** [review]

File: Crypto_App_iOS/Flows/Screens/Home/HomeView.swift (hunk 3 above)
Comment (zhenyapavlovsky) [reply Crypto_App_iOS/Flows/Screens/Home/HomeView.swift:65]: Done

File: Crypto_App_iOS/Flows/Screens/Home/Data.swift (hunk 4 above)
Comment (zhenyapavlovsky) [reply Crypto_App_iOS/Flows/Screens/Home/Data.swift:19]: Done

**Pull request merged**